config/main.py

```diff
@@ -733,7 +733,9 @@ def _get_sonic_services():
 
 def _get_delayed_sonic_services():
     out = clicommon.run_command("systemctl list-dependencies --plain sonic-delayed.target | sed '1d'", return_cmd=True)
-    return (unit.strip().rstrip('.timer') for unit in out.splitlines())
+    timers = [unit for unit in out.splitlines() if(clicommon.run_command("systemctl is-enabled {}".format(unit),
```

Comment:
>unit [](http://example.com/codeflow?start=14&length=4)

This statement is too complex. Could you rewrite in traditional control flow, like for-loop or if-block?